src/confcom/HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 
 Release History
 ===============
+0.3.5
```

Comment:
please update version as bot recommended.

Reply:
Hi, we are waiting for a particular feature to be ready and merged to make a 1.0.0 version. Thanks!